src/content/docs/merge-queue/performance.mdx

```diff
@@ -43,33 +43,33 @@ Based on this trade-off, there are 3 scenarios that you can choose from:
   time.
 
 - **Reliable and Fast:** If you aim to reduce latency and maximize throughput
-  without considering CI costs, enable **[speculative
-  checks](/merge-queue/speculative-checks)**. This feature lets Mergify test
-  pull requests in parallel, predicting potential merges and executing CI runs
-  simultaneously. Speculative checks can slightly increase CI cost: there might
-  be certain scenarios that are tested and whose results won't be used if a pull
+  without considering CI costs, enable **[parallel speculative
+  checks](/merge-queue/parallel-checks)**. This feature lets Mergify test pull
+  requests in parallel, predicting potential merges and executing CI runs
+  simultaneously. Parallel checks can slightly increase CI cost: there might be
+  certain scenarios that are tested and whose results won't be used if a pull
   request ahead in the queue fails.
 
 - **Cheap and Fast:** By using [batch mode](/merge-queue/batches), you can merge
   groups of pull requests simultaneously. This reduces CI runs but merges pull
   request that are not tested individually. There is therefore a theoretical
   risk than a hidden failure is merged, while the whole batch passes the CI.
   
-You can also combine speculative checks and batching to achieve a balanced
-approach between reliability, cheapness, and velocity. This configuration
-offers a mix of both strategies, allowing you to test multiple batches of
-requests concurrently.
+You can also combine speculative parallel checks and batching to achieve a
```

Comment:
Here, you use "speculative parallel checks", while you use "parallel speculative checks" above. We should choose one.